File: app/src/main/java/chat/rocket/android/fragment/chatroom/HomeFragment.java
package chat.rocket.android.fragment.chatroom;

import chat.rocket.android.R;

public class HomeFragment extends AbstractChatRoomFragment {

  public HomeFragment() {}

  @Override
  protected int getLayout() {
    return R.layout.fragment_home;
  }

  @Override
  protected void onSetupView() {
    setToolbarTitle(getText(R.string.fragment_home_title));
  }
}

File: app/src/main/java/chat/rocket/android/fragment/chatroom/HomeFragment.kt
package chat.rocket.android.fragment.chatroom

import chat.rocket.android.R

class HomeFragment : AbstractChatRoomFragment() {

    override fun getLayout(): Int {
        return R.layout.fragment_home
    }

    override fun onSetupView() {
        setToolbarTitle(getText(R.string.home_fragment_title))
    }
}